Library/Formula/wellington.rb

```diff
@@ -17,35 +17,58 @@ class Wellington < Formula
 
   depends_on "go" => :build
   depends_on "pkg-config" => :build
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
 
-  go_resource "github.com/wellington/spritewell" do
-    url "https://github.com/wellington/spritewell.git",
-        :revision => "748bfe956f31c257605c304b41a0525a4487d17d"
+  stable do
+    depends_on "libsass"
   end
 
-  go_resource "github.com/go-fsnotify/fsnotify" do
-    url "https://github.com/go-fsnotify/fsnotify.git",
-        :revision => "f582d920d11386e8ae15227bb5933a8f9b4c3dec"
+  devel do
+    url "https://github.com/wellington/wellington/archive/c574754c39806e8c52682f15f49c8ff819d0f962.tar.gz"
+    sha256 "92c5d4bdd6260f96aed0e20fa06c6200b1c57281dbd32dbd00b1c06636ef6e10"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
```

Comment:
This does not work, the "-X some long option" is a single string argument, separating it breaks the parameters.

changing to

    system "go", "build", "-ldflags", "-X", "main.version", "#{ENV["VERSION"]}", "-o", "dist/wt", "wt/main.go"

resulted in Go build errors
```
go build -ldflags -X main.version v0.8.1 -o dist/wt wt/main.go
	/usr/local/go/src/dist/wt (from $GOROOT)
	/private/var/folders/9j/wby800x10t76371jxq8hwjd80000gp/T/wellington20150517-2746-htrrv7/wellington-0.8.1/src/dist/wt (from $GOPATH)
can't load package: package wt/main.go: cannot find package "wt/main.go" in any of:
	/usr/local/go/src/wt/main.go (from $GOROOT)
	/private/var/folders/9j/wby800x10t76371jxq8hwjd80000gp/T/wellington20150517-2746-htrrv7/wellington-0.8.1/src/wt/main.go (from $GOPATH)
couldn't understand kern.osversion `14.3.0'
```